src/sass/_header.scss

```diff
@@ -0,0 +1,10 @@
+.head {
+    width: 100%;
+    height: 502px;
+    background: #FFB8CA;
```

Comment:
w przyszłości trzeba będzie to na zmienną sass'ową zamienić, ale rozumiem, że na razie ten scss jest na próbe